pkg/oc/cli/importimage/importimage.go

```diff
@@ -326,11 +360,11 @@ func (o *ImportImageOptions) importAll(stream *imageapi.ImageStream) (*imageapi.
 	return o.newImageStreamImportAll(stream, from), nil
 }
 
-func (o *ImportImageOptions) importTag(stream *imageapi.ImageStream) (*imageapi.ImageStreamImport, error) {
+func (o *ImportImageOptions) importTag(stream *imageapiv1.ImageStream) (*imageapiv1.ImageStreamImport, error) {
 	from := o.From
 	tag := o.Tag
 	// follow any referential tags to the destination
-	finalTag, existing, multiple, err := imageapi.FollowTagReference(stream, tag)
+	finalTag, existing, multiple, err := followTagReferenceV1(stream, tag)
```

Comment:
I hate to say that, but create a shim, and don't copy. At some point in time some victim (in my unclear memory I recall I volunteered) will fix the controller to use the externals as well and we'll just remove the shim.

Reply:
I am having the same problem here, although the helper `pkg/image/apis/image/v1.Convert_v1_ImageStream_To_image_ImageStream` does not depend on a conversion scope, on of the conversion funcs that _it_ depends on does: https://github.com/openshift/origin/blob/master/pkg/image/apis/image/v1/conversion.go#L160